Jellyfin.Api/Controllers/ImageController.cs

```diff
@@ -1,4 +1,3 @@
-using System;
```

Comment:
^ I think this removal doesn't really fix the build

Reply:
Yeah, this is the cause of all these errors:
`The type or namespace name 'Guid' could not be found (are you missing a using directive or an assembly reference?)`